test/functional/tools/composite_shapefile.xml

```diff
@@ -0,0 +1,35 @@
+<tool id="composite" name="composite" version="1.0.0">
```

Comment:
To test the tool you need to add it to `test/functional/tools/samples_tool_conf.xml` and the datatype to `test/functional/tools/sample_datatypes_conf.xml`.